src/amo/reducers/reviews.js

```diff
@@ -429,6 +461,18 @@ export default function reviewsReducer(
         },
       });
     }
+    case FETCH_REVIEWS: {
+      const {
```

Comment:
nit: this looks like it could fit on one line

It's a bummer that prettier won't collapse short lines automatically.

Reply:
If you put it all on one line, prettier will leave it. If you break any part of it on to a new line, prettier will break the whole thing. It's a weird feature. I wish it were consistent.

Reply:
I did put it all on one line, and prettier still changed it for me. It's doing the change automatically in my IDE, so maybe that behaves differently from just using the command line. With my IDE, it will not let me put it all on one line.